ejc-sql.el

```diff
@@ -111,6 +111,14 @@ results. When nil, otherwise, provide `ejc-sql' users expected behaviour."
   :group 'ejc-sql
   :type '(plist :key-type string :value-type (vector symbol string)))
 
+(defcustom ejc-completion-system 'ido
```

Comment:
I think the `'default` should be default :)

Reply:
I was hesitant to change because it affects the current default behavior. Perhaps we can use the name to `standard`, or make a note of the change in the README?